src/k3s/mod.rs

```diff
@@ -224,6 +222,84 @@ mod tests {
         Ok(())
     }
 
+    // Based on: https://github.com/kube-rs/kube/blob/main/examples/pod_api.rs
+    #[tokio::test]
+    async fn pod_api() -> Result<(), Box<dyn std::error::Error + 'static>> {
+        let conf_dir = temp_dir();
+        let k3s = K3s::default()
+            .with_conf_mount(&conf_dir)
+            .with_privileged(true)
+            .with_userns_mode("host");
+
+        let k3s_container = k3s.start().await?;
+
+        let client = get_kube_client(&k3s_container).await?;
+
+        // Manage pods
+        let pods: Api<Pod> = Api::default_namespaced(client);
+
+        // Create Pod blog
+        let p: Pod = serde_json::from_value(json!({
+            "apiVersion": "v1",
+            "kind": "Pod",
+            "metadata": { "name": "blog" },
+            "spec": {
+                "containers": [{
+                  "name": "blog",
+                  "image": "clux/blog:0.1.0"
+                }],
+            }
+        }))?;
```

Comment:
I'd suggest to move this into [/examples](https://github.com/testcontainers/testcontainers-rs-modules-community/tree/main/examples) instead of tests. Because it uses some custom image reference and it would be extra waiting time and load for CI

Reply:
are examples execute in ci as well? in a later stage, somewhere?

Reply:
I am asking, because a sample that does not run is not worth it imo

Reply:
No, examples are compiled but not executed. They mostly needed as a demonstration - but yes, that's valid point.

I just don't want CI to depend on some random image (e.g `clux/blog:0.1.0`). We can add the test, but with some more predictable image

Reply:
> I just don't want CI to depend on some random image

understood.

how about re-using something like the postgres image - that's being used anyway? I'll have a look at it! or maybe some very small minimal image.